### ISSUE TITLE
Skript Fails to Create its Files

### ISSUE THREAD

**Issue by OfficialDonut** [SkriptLang/Skript#1755]
### Description
Skript fails to create its files if you delete everything except the scripts folder

### Steps to Reproduce
Step 1) delete everything in Skript folder except scripts folder
Step 2) start server

### Expected Behavior
Skript's files should be created (only the materials.json is created)

### Errors / Screenshots
```
[Skript] Task #2 for Skript v2.3-beta4 generated an exception
ch.njol.skript.SkriptAPIException: Cannot load variables before the config
        at ch.njol.skript.variables.Variables.load(Variables.java:120) ~[?:?]
        at ch.njol.skript.Skript$2.run(Skript.java:491) ~[?:?]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:76) ~[spigot-1.13.2-R0.1-SNAPSHOT-b1976.jar:git-Spigot-e4d0c66-aed3aec]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:361) [spigot-1.13.2-R0.1-SNAPSHOT-b1976.jar:git-Spigot-e4d0c66-aed3aec]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:893) [spigot-1.13.2-R0.1-SNAPSHOT-b1976.jar:git-Spigot-e4d0c66-aed3aec]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2-R0.1-SNAPSHOT-b1976.jar:git-Spigot-e4d0c66-aed3aec]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot-1.13.2-R0.1-SNAPSHOT-b1976.jar:git-Spigot-e4d0c66-aed3aec]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot-1.13.2-R0.1-SNAPSHOT-b1976.jar:git-Spigot-e4d0c66-aed3aec]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
```

### Server Information
* **Server version/platform:** Spigot 1.13.2
* **Skript version:** Skript v2.3-beta4

**Comment by Nicofisi** [comment]
Pretty sure @TheLimeGlass reported this once? Or maybe it was only on Discord? hmmm..

**Comment by bensku** [comment]
Can't find it, so let's keep this.

**Comment by Matocolotoe** [comment]
Can confirm, happens with latest version.

**Comment by ShaneBeee** [comment]
Closing since this was fixed in #2127
